hi-ml/src/health_ml/lightning_container.py

```diff
@@ -183,6 +183,21 @@ def get_hyperdrive_config(self) -> Optional[HyperDriveConfig]:
             return self.get_different_seeds_hyperdrive_config()
         return None
 
+    def get_hyperparam_args(self) -> Optional[Dict[str, Any]]:
+        """
+        Returns a dictionary of hyperparameter search arguments that will be passed to an AML v2 command to
+        enable either hyperparameter tuning,  cross validation, or running with different seeds.
+
+        :return: A dictionary of hyperparameter search arguments and values.
+        """
+        if self.hyperdrive:
```

Comment:
hyperdrive is not available with the new sdk only right? How about checking here that strictly_aml_v1 is False and suggesting to switch to the sdk v1 for hyperdrive runs?